packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -259,16 +260,15 @@ class VsCode {
   String toString() =>
       'VS Code ($version)${_extensionVersion != Version.unknown ? ', Flutter ($_extensionVersion)' : ''}';
 
-  static String _getVersionFromPackageJson(String packageJsonPath) {
-    if (!globals.fs.isFileSync(packageJsonPath)) {
+  static String _getVersionFromPackageJson(String packageJsonPath, FileSystem fileSystem) {
+    if (!fileSystem.isFileSync(packageJsonPath)) {
       return null;
     }
-    final String jsonString = globals.fs.file(packageJsonPath).readAsStringSync();
+    final String jsonString = fileSystem.file(packageJsonPath).readAsStringSync();
     try {
       final Map<String, dynamic> jsonObject = castStringKeyedMap(json.decode(jsonString));
       return jsonObject['version'] as String;
-    } on FormatException catch (err) {
-      globals.printTrace('Error parsing VSCode $packageJsonPath:\n$err');
```

Comment:
Could this still be useful?

Reply:
I was only using this as a sentinel in a test, but not throwing an exception accomplishes the same goal